### ISSUE TITLE
Is my follow button spoiled?

### ISSUE THREAD

**Issue by anonymouslyfen** [CommanderRoot/twitch-tools#102]
Hi, my twitch account was recently raided by bots and it caused my twitch account to be abnormal. So I used commanderroot as a form of solution to block the bots but it keeps automatically blocking the wrong person which is not a bot.

For example, my friend Nyxediaa want to follow me but the follow button is blocked as per the video, and once she click on the follow after i unblock her, it shows she is on the block list.

**Comment by anonymouslyfen** [comment]
[https://drive.google.com/file/d/1rSnvYs6kKy1PffIVsCSB3LZuHbMbn-mu/view?usp=sharing](url)

**Comment by CommanderRoot** [comment]
Duplicate of #30

The account `nyxediaa` was added to the known bot list on `2022-08-04` as it was part of one or multiple follow bot raids. It's quite possible that it was compromised at that time.
Either way it happend I have now removed it from the list.
